slack.md

```diff
@@ -1,147 +1,65 @@
 # Slack guidelines
 
-Hi and welcome to Receipt Bank!
+This page will acquaint you with how we use Slack in Receipt Bank. It's organized by topic. Please read it carefully and follow the instructions.
```

Comment:
We can also drop the "and follow the instructions". Seems a bit too commanding.

Reply:
Agreed